### PR TITLE
docs: add non-flakes quick start documentation

### LINKED ISSUE #64: Add non-Flakes Instructions to Quick Start
Some people don't use flakes and should have documentation as well. :)

### PR DESCRIPTION
**Pull request by RaitoBezarius**
Here we go, closes #64.

### CODE REVIEW

File: docs/QUICK_START.md [hunk 1]
```diff
@@ -119,6 +119,48 @@ the ESP. `systemd-boot` will display these during boot. This can be
...
+  Writing new sources file
+Done: Adding package lanzaboote
+```
+
+Below is a fragment of a NixOS configuration that enables the SecureBoot stack.
+
+```nix
+{ pkgs, lib, ... }:
+let sources = import ./nix/sources.nix;
+    lanzaboote = import sources.lanzaboote;
```
Comment (nikstur) [inline docs/QUICK_START.md:138]: Please ignore this nit if you hate this formatting :D

```suggestion
let 
    sources = import ./nix/sources.nix;
    lanzaboote = import sources.lanzaboote;
```

**Review by nikstur — APPROVED** [review]
Even though only users that have enough experience of NixOS should even use Lanzaboote, I still think it would be nice to add a small fragment where you show how to import the module into your `configuraton.nix`.

**Comment by RaitoBezarius** [comment]
> Nice!
> 
> Even though only users that have enough experience of NixOS should even use Lanzaboote, I still think it would be nice to add a small fragment where you show how to import the module into your `configuraton.nix`.

Isn't this already the case? Or do you mean importing the file referenced in the quick start?

**Comment by nikstur** [comment]
Ah now I see it. This fragment is supposed to be `configuration.nix` itself. Maybe you can just add a comment to the top of the block. For example:  # file: `configuration.nix`

**Comment by RaitoBezarius** [comment]
> Ah now I see it. This fragment is supposed to be `configuration.nix` itself. Maybe you can just add a comment to the top of the block. For example: `# file: configuration.nix`

Done

**Comment by nikstur** [comment]
Awesome!

**Pull request merged**